pkg/controller/installation/products/rhsso/reconciler.go

```diff
@@ -16,37 +16,66 @@ import (
 	"k8s.io/api/core/v1"
 	k8serr "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/kubernetes"
 	pkgclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 var (
-	defaultInstallationNamespace = "rhsso"
-	keycloakName                 = "rhsso"
-	keycloakRealmName            = "openshift"
+	DefaultRhssoNamespace = "rhsso"
```

Comment:
I don't think this should be public, other reconcilers can get this value from the RHSSO config.

Reply:
Yeah my bad. I'll update this